pkgs/tools/security/pinentry/default.nix

```diff
@@ -66,6 +67,7 @@ pinentryMkDerivation rec {
   ];
 
   configureFlags = [
+    "--disable-doc" # disabling docs because pinentry doc build is broken
```

Comment:
Could you share the error? Maybe we can fix it.

Reply:
The solution is to to this pinentry generation was that maintainer mode wasn't activated, leaving the `missing` script out of execution. https://autotools.io/automake/maintainer.html